frontend/src/components/GddTransactionList.vue

```diff
@@ -110,6 +117,10 @@ export default {
       })
       window.scrollTo(0, 0)
     },
+    previousBookedBalance(idx) {
```

Comment:
i consider this a bit hacky - tho its fine for now. In my opinion we should have a reference to the previous transaction instead of only the balance value if we do stuff like this.
Furthermore this brings more logic into the frontend - but I guess thats inevitable for this purpose if we do not want to transfer data twice